app/main.py

```diff
@@ -1,5 +1,81 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: float) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: float,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
```suggestion
        self.coords = coords or [0,0]
```
Apply this for Flying robot as well